{
    "data_folder_path": "data/CVE-2023-48699",
    "pull_requests_urls": [
        "https://github.com/ubertidavide/fastbots/pull/3#issue-2003080806"
    ],
    "cve_name": "CVE-2023-48699",
    "repo": "ubertidavide/fastbots",
    "warnings": {
        "cve": "CVE-2023-48699",
        "warnings": []
    },
    "files_extension_stats": {
        ".md": 1,
        ".toml": 1,
        ".py": 4
    },
    "file_number": 6,
    "removed_files_number": 0,
    "added_files_number": 0,
    "nb_lines_affected": 117,
    "private_key_files": [],
    "languages": [
        "Python"
    ]
}